README.md

```diff
@@ -354,18 +334,17 @@ collection names) in MongoDB (see <a href="http://docs.mongodb.org/manual/refere
 for further information). Related to this, the STH generates the collection names using 2 possible mechanisms:
 
 1. <u>Plain text</u>: In case the `SHOULD_HASH` configuration parameter is set to 'false', the collection names are
-generated as a concatenation of the `COLLECTION_PREFIX` plus the service path (in case of the collection-per-service-path
-data model) plus the entity id plus the entity type (in case of the collection-per-entity data model) plus the attribute name
-(in case of the collection-per-attribute data model) plus '.aggr' for the collections of the aggregated data. The length
-of the collection name plus the `DB_PREFIX` plus the database name (or service) should not be more than 120 bytes using UTF-8
-format or MongoDB will complain and will not create the collection, and consequently no data would be stored by the STH.
+generated as a concatenation of the `COLLECTION_PREFIX` plus the service path plus the entity id plus the entity type
+plus '.aggr' for the collections storing the aggregated data. The length of the collection name plus the `DB_PREFIX` plus
+the database name (or service) should not be more than 120 bytes using UTF-8 format or MongoDB will complain and will not
+create the collection, and consequently no data would be stored by the STH. A warning message is logged in case this happens.
 
 2. <u>Hash based</u>: In case the `SHOULD_HASH` option is set to something distinct from 'false' (the default option), the
 collection names are generated as a concatenation of the `COLLECTION_PREFIX` plus a generated hash plus '.aggr' for the
 collections of the aggregated data. To avoid collisions in the generation of these hashes, they are forced to be 20 bytes
 long at least. Once again, the length of the collection name plus the `DB_PREFIX` plus the database name (or service) should not
 be more than 120 bytes using UTF-8 or MongoDB will complain and will not create the collection, and consequently no data
-would be stored by the STH. The hash function used is SHA-512.
+would be stored by the STH. The hash function used is SHA-512. A warning message is logged in case this happens.
 
 In case of using hashes as part of the collection names and to let the user or developer easily recover this information,
 a collection named ```DB_COLLECTION_PREFIX + _collection_name``` is created and fed with information regarding the mapping
```

Comment:
Fixed in https://github.com/telefonicaid/IoT-STH/commit/bbc8d1837e3ac7f44995e2f9f0f786a309336e1d